<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QDialog" name="Main">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2516</width>
    <height>1600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>2516</width>
    <height>1600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>IM혜민 Bold</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainView</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(242, 242, 242, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QLabel" name="logo_mnvision">
   <property name="geometry">
    <rect>
     <x>-50</x>
     <y>20</y>
     <width>461</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/img/img/엠엔비전.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>1270</y>
     <width>1921</width>
     <height>71</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>281</width>
     <height>31</height>
    </size>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1901</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="current_time_label">
       <property name="font">
        <font>
         <family>IM혜민 Bold</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>00:00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="playing_silder">
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="singleStep">
        <number>4</number>
       </property>
       <property name="pageStep">
        <number>11</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
       <property name="sliderPosition">
        <number>3</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="video_total_label">
       <property name="font">
        <font>
         <family>IM혜민 Bold</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>00:00</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>1360</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>167</width>
     <height>41</height>
    </size>
   </property>
   <widget class="QPushButton" name="video_before_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>IM혜민 Bold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources/resources.qrc">
      <normaloff>:/img/img/rewind (1).png</normaloff>:/img/img/rewind (1).png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="video_stop_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>IM혜민 Bold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>❚❚</string>
    </property>
   </widget>
   <widget class="QPushButton" name="video_after_button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>IM혜민 Bold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources/resources.qrc">
      <normaloff>:/img/img/fast-forward (1).png</normaloff>:/img/img/fast-forward (1).png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="Log_list">
   <property name="geometry">
    <rect>
     <x>2080</x>
     <y>800</y>
     <width>401</width>
     <height>401</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>131</width>
     <height>171</height>
    </size>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="video_frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>2050</width>
     <height>1150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>2050</width>
     <height>1150</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="directory_botton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>1270</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/img/img/upload (1).png</normaloff>:/img/img/upload (1).png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="img_log">
   <property name="geometry">
    <rect>
     <x>2080</x>
     <y>360</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/img/img/로그.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="setting_button">
   <property name="geometry">
    <rect>
     <x>2080</x>
     <y>1230</y>
     <width>191</width>
     <height>111</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>101</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>IM혜민 Bold</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: #ffffff;
selection-color: #ffffff;
color: #ffffff;

border: 2px solid  #000000;
border-radius: 25px;
padding-right: 5px; 
padding-left: 5px;
padding-top: 5px;
padding-bottom: 5px;
}

QPushButton:hover
{
	background-color: #F28B30;
	border: 1px solid  #000000;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/img/img/설정.png</normaloff>:/img/img/설정.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="img_video_info">
   <property name="geometry">
    <rect>
     <x>2080</x>
     <y>90</y>
     <width>281</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/img/img/영상 정보.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>2080</x>
     <y>480</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCalendarWidget QWidget{
background-color:white;

}

QCalendarWidget QToolButton{
background-color:white;
color: black;
icon-size: 30px;
}

QCalendarWidget QAbstractItemView:enabled{
background-color: #FAF5F7;
color: black;
}


</string>
   </property>
   <property name="date">
    <date>
     <year>2024</year>
     <month>12</month>
     <day>1</day>
    </date>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>2300</x>
     <y>480</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>case all</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>case helmet</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>case car</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>case rack</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="search">
   <property name="geometry">
    <rect>
     <x>2430</x>
     <y>480</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>KoPub돋움체 Bold</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/img/img/검색_24.png</normaloff>:/img/img/검색_24.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QTextBrowser" name="video_info_label">
   <property name="geometry">
    <rect>
     <x>2080</x>
     <y>180</y>
     <width>391</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;(미리보기)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="drawing_button">
   <property name="geometry">
    <rect>
     <x>2290</x>
     <y>1230</y>
     <width>191</width>
     <height>111</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>101</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>IM혜민 Bold</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: #ffffff;
selection-color: #ffffff;
color: #ffffff;

border: 2px solid  #000000;
border-radius: 25px;
padding-right: 5px; 
padding-left: 5px;
padding-top: 5px;
padding-bottom: 5px;
}

QPushButton:hover
{
	background-color: #F28B30;
	border: 1px solid  #000000;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/img/img/구역지정_50.png</normaloff>:/img/img/구역지정_50.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>135</width>
     <height>138</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="rec_film">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>2050</width>
     <height>1150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>2050</width>
     <height>1150</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>img_log</zorder>
  <zorder>logo_mnvision</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox</zorder>
  <zorder>Log_list</zorder>
  <zorder>directory_botton</zorder>
  <zorder>setting_button</zorder>
  <zorder>img_video_info</zorder>
  <zorder>video_frame</zorder>
  <zorder>dateTimeEdit</zorder>
  <zorder>comboBox</zorder>
  <zorder>search</zorder>
  <zorder>video_info_label</zorder>
  <zorder>drawing_button</zorder>
  <zorder>rec_film</zorder>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>setting_button</sender>
   <signal>clicked()</signal>
   <receiver>Main</receiver>
   <slot>go_setting()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>2080</x>
     <y>1340</y>
    </hint>
    <hint type="destinationlabel">
     <x>2074</x>
     <y>1478</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>directory_botton</sender>
   <signal>clicked()</signal>
   <receiver>Main</receiver>
   <slot>load_video()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>1317</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>1373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>video_stop_button</sender>
   <signal>clicked()</signal>
   <receiver>Main</receiver>
   <slot>video_pause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1029</x>
     <y>1410</y>
    </hint>
    <hint type="destinationlabel">
     <x>1115</x>
     <y>1498</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>video_before_button</sender>
   <signal>clicked()</signal>
   <receiver>Main</receiver>
   <slot>move_before_10s()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>948</x>
     <y>1406</y>
    </hint>
    <hint type="destinationlabel">
     <x>944</x>
     <y>1508</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>video_after_button</sender>
   <signal>clicked()</signal>
   <receiver>Main</receiver>
   <slot>move_after_10s()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1112</x>
     <y>1410</y>
    </hint>
    <hint type="destinationlabel">
     <x>1090</x>
     <y>1475</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drawing_button</sender>
   <signal>clicked()</signal>
   <receiver>Main</receiver>
   <slot>go_drawing()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>2354</x>
     <y>1316</y>
    </hint>
    <hint type="destinationlabel">
     <x>2353</x>
     <y>1437</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>search</sender>
   <signal>clicked()</signal>
   <receiver>Main</receiver>
   <slot>log_search()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>2470</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>2407</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>go_setting()</slot>
  <slot>showlist()</slot>
  <slot>video_FileLoad()</slot>
  <slot>slot1()</slot>
  <slot>load_video()</slot>
  <slot>video_pause()</slot>
  <slot>move_before_10s()</slot>
  <slot>move_after_10s()</slot>
  <slot>go_drawing()</slot>
  <slot>slot2()</slot>
  <slot>log_search()</slot>
 </slots>
</ui>
